### DISCUSSION TITLE
잠시 후 다시 시도하세요. issues documentation 생성

### DISCUSSION THREAD

**Discussion by OneTop4458** [changdoc/naver-vaccine-macro#41] [Ideas]
잠시 후 다시 시도하세요 발생에 대한 원인 및 해결 방안을 아래와 같은 형태로 설명을 올리는 것을 건의 해 봅니다.
https://github.com/changdoc/naver-vaccine-macro/issues/29

최근 쓰레드 댓글들을 보니 RTFM을 안 보는 사람이 너무 많은 듯 합니다.
RTFM: Read The Fucking Manual (빌어먹을 안내서를 읽으시오)

**Comment by changdoc** [comment]
네 한번 정리해두는게 좋을듯 하네요
